app/users/page.tsx

```diff
@@ -1,59 +1,34 @@
-'use client';
-import React from 'react';
+'use client'
```

Comment:
Lets see if we can render this in the backend by removing `use client`

Reply:
Unable to render without due to useState from booking

Reply:
The react-charts is also a client side render unfortunately this was part of the reason I could not get it to display before